components/feature/toolbar/src/test/java/mozilla/components/feature/toolbar/WebExtensionToolbarTest.kt

```diff
@@ -4,53 +4,81 @@
 
 package mozilla.components.feature.toolbar
 
+import android.graphics.Bitmap
 import android.graphics.Color
-import android.graphics.drawable.Drawable
+import android.graphics.drawable.BitmapDrawable
 import android.view.View
 import android.widget.ImageView
 import android.widget.LinearLayout
 import android.widget.TextView
 import androidx.test.ext.junit.runners.AndroidJUnit4
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.setMain
 import mozilla.components.concept.engine.webextension.BrowserAction
 import mozilla.components.support.base.android.Padding
+import mozilla.components.support.test.argumentCaptor
+import mozilla.components.support.test.ext.joinBlocking
 import mozilla.components.support.test.mock
 import mozilla.components.support.test.robolectric.testContext
 import mozilla.components.support.test.whenever
+import org.junit.After
 import org.junit.Assert.assertEquals
 import org.junit.Assert.assertFalse
 import org.junit.Assert.assertTrue
+import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.mockito.Mockito.verify
 
 @RunWith(AndroidJUnit4::class)
 class WebExtensionToolbarTest {
+    private val testDispatcher = TestCoroutineDispatcher()
+
+    @Before
+    @ExperimentalCoroutinesApi
+    fun setUp() {
```

Comment:
nit: I think we could replace this block with [MainCoroutineRule](https://github.com/mozilla-mobile/android-components/blob/9b8e27603efea171c8f3764de9a25cc3f1e84ec9/components/support/test/src/main/java/mozilla/components/support/test/rule/MainCoroutineRule.kt#L28)

Reply:
Ah, nice! Updated.